package.json

```diff
@@ -5,10 +5,11 @@
   "main": "skin-deep.js",
   "typings": "skin-deep.d.ts",
   "scripts": {
-    "test": "mocha",
     "lint": "eslint . --max-warnings 0",
     "coverage": "istanbul cover _mocha --",
-    "doc": "doctoc README.md"
+    "doc": "doctoc README.md",
+    "test": "mocha",
+    "test:all": "REACT=0.14 sh ./install-relevant-react.sh && npm run test && REACT=15.4 sh ./install-relevant-react.sh && npm run test && REACT=15.5 sh ./install-relevant-react.sh && npm run test"
```

Comment:
Do you think it's worth pulling this out into a script which reads the list of versions from `.travis.yml` ?

I'm not sure, might be neater going forwards if React keeps playing around with these APIs